### PR TITLE
Add support for limit, order and type arguments

### PR DESCRIPTION
**Pull request by weskamm**
As the title says. Added a cURL example showing the new parameters in action.

@hwbllmnn please review

### CODE REVIEW

File: src/main/kotlin/de/terrestris/geoserver/wps/DistinctValues.kt [hunk 1]
```diff
@@ -42,6 +42,7 @@ import org.geotools.process.factory.DescribeParameter
 import org.geotools.process.factory.DescribeProcess
 import org.geotools.process.factory.DescribeResult
 import org.geotools.util.logging.Logging
+import org.geotools.xml.xsi.XSISimpleTypes
```
Comment (hwbllmnn) [inline src/main/kotlin/de/terrestris/geoserver/wps/DistinctValues.kt]:
```suggestion
```

**Review by hwbllmnn — APPROVED** [review]
Nice one, please merge!

File: src/main/kotlin/de/terrestris/geoserver/wps/DistinctValues.kt [hunk 2]
```diff
@@ -117,11 +121,21 @@ class DistinctValues(private val geoServer: GeoServer) : GeoServerProcess {
                 }
                 LOGGER.fine("Custom SQL: $sql")
             }
+            if (order != null && (order.toLowerCase() == "asc" || order.toLowerCase() == "desc")) {
+                sql += " ORDER BY $propertyName $order"
+            }
+            if (limit != null) {
+                sql += " LIMIT $limit"
+            }
```
Comment (marcjansen) [inline src/main/kotlin/de/terrestris/geoserver/wps/DistinctValues.kt:127]: Wouldn't it make sense to now log the SQL on finest level, and not some lines above. I am only looking at the diff, so might be wrong

**Review by marcjansen — APPROVED** [review]

File: src/main/kotlin/de/terrestris/geoserver/wps/DistinctValues.kt (hunk 2 above)
Comment (weskamm) [reply src/main/kotlin/de/terrestris/geoserver/wps/DistinctValues.kt:127]: right, enhanced

**Comment by weskamm** [comment]
Thanks for reviews

**Pull request merged**